File: guanabara-python/src/lista03/exercicio28.py
largura = float(input("Insira a largura do terreno: "))
comprimento = float(input("Insira o comprimento do terreno: "))
area = largura * comprimento

if area < 100:
    print(f"Área: {area}. Tipo do terreno: Popular")
elif area >= 100 and area < 500:
    print(f"Área: {area}. Tipo do terreno: Master")
else:
    print(f"Área: {area}. Tipo do terreno: VIP")

File: guanabara-c/lista03/exercicio28.c
#include <stdio.h>

int main(){

    float largura, comprimento, area;

    printf("Insira a largura do terreno: ");
    scanf("%f", &largura);
    printf("Insira o comprimento: ");
    scanf("%f", &comprimento);

    area = largura * comprimento;

    if (area < 100){
        printf("Área: %.2f m². TERRENO POPULAR.", area);
    }
    else if (area >= 100 && area < 500){
        printf("Área: %.2f m². TERRENO MASTER.", area);
    }
    else{
        printf("Área: %.2f m². TERRENO VIP.", area);
    }

    return 0;
}
